### PR TITLE
implement holiday crd

### PR DESCRIPTION
**Pull request by jaotors**
capture

### CODE REVIEW

File: app/Http/Controllers/HolidayController.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+     * @param  \Illuminate\Http\Request $request
+     *
+     * @return \Illuminate\Http\Response $response
+     */
+
+    public function store(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'name' => 'required',
+            'date' => 'required'
```
Comment (jmramos02) [inline app/Http/Controllers/HolidayController.php]: Add validation for "date"

File: resources/views/admin/holiday/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                        <tr>
+                            <th>Name</th>
+                            <th>Date</th>
+                            <th>Action</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach($holidays as $holiday)
+                            <tr>
+                                <td>{{$holiday->name}}</td>
```
Comment (jmramos02) [inline resources/views/admin/holiday/index.blade.php]: You forgot to display the date of the holiday 👍

File: resources/views/admin/layout.blade.php [hunk 3]
```diff
@@ -23,6 +23,7 @@
             <li><a href="/admin/departments"><span class="glyphicon glyphicon-th-list" aria-hidden="true"></span>Courses</a></li>
             <li><a href="/admin/categories"><span class="glyphicon glyphicon-duplicate" aria-hidden="true"></span>Categories</a></li>
             <li><a href="/admin/books"><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Books</a></li>
+            <li><a href="/admin/holidays"><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Holidays</a></li>
```
Comment (jmramos02) [inline resources/views/admin/layout.blade.php]: don't we have any calendar icon?

File: resources/views/admin/holiday/index.blade.php (hunk 2 above)
Comment (jaotors) [inline resources/views/admin/holiday/index.blade.php]: done with this (y)

**Pull request merged**